SU-SWS/stanford_publication
# D8CORE-3951: adding margin bottom to the filtered list of publications

# READY FOR REVIEW

# Summary
- This adds the margin bottom to the filtered publications list.
- This fixes the narrow widths when there is little content in a publication.
- This fixes the pagination back to the infinite scroll and Load More button.

# Needed By (Date)
- 5/5

# Urgency
- high

# Steps to Test

1. Pull in this change and clear the caches.
2. Pull in [PR 41](https://github.com/SU-SWS/stanford_publication/pull/41) and clear caches.
2. Go to your publications page and generate 20 + pages.
3. Verify there is a margin bottom.
4. Note the pagination on the filtered pages is known by design. 

# Affected Projects or Products
- Publications

# Associated Issues and/or People
- [D8CORE-3951](https://stanfordits.atlassian.net/browse/D8CORE-3951)
- https://github.com/SU-SWS/stanford_publication/pull/41

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
